package.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "aem-site-template-basic",
   "version": "1.0.0",
-  "description": "Basic AEM site template",
+  "description": "Basic AEM site template that serves as a clean starting point.",
   "license": "MIT License, Copyright 2020 Adobe Systems Incorporated",
   "scripts": {
     "build": "aem-site-template-builder",
     "deploy": "curl -u admin:admin -F 'file=@site-template.zip' http://localhost:4502/conf/global/site-templates.import.html"
   },
   "devDependencies": {
     "@adobe/aem-site-template-builder": "0.2.0"
+  },
+  "site_template": {
```

Comment:
Looking [at the spec](https://docs.npmjs.com/cli/v6/configuring-npm/package-json) and seeing the usage of camel case like `devDependencies `, I would propose to rename to `siteTemplate` for consistency reasons.